modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDisplayPageTemplates.testcase

```diff
@@ -387,6 +387,132 @@ definition {
+			JSONCategory.addCategory(
+				groupName = "Global",
+				title = "Topic Category",
+				vocabularyName = "Topic");
+		}
+
+		task ("And some categories of the new vocabulary are selected inside a product (e.g. ABS Sensor)") {
+			CommerceNavigator.gotoPortlet(
+				category = "Product Management",
+				portlet = "Products");
+
+			Click(
+				key_entryName = "ABS Sensor",
+				locator1 = "CommerceEntry#TABLE_LIST_TITLE");
+
+			Click(
+				key_fieldName = "Test Vocabulary",
+				locator1 = "Button#SELECT_BUTTON");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_BODY");
+
+			Click.clickAtNotVisible(
+				key_nodeName = "Category 1",
+				locator1 = "Treeview#NODE_CHECKBOX");
+
+			SelectFrame(value1 = "relative=top");
+
+			Click(locator1 = "Button#DONE");
+		}
+
+		task ("And the category related to Topic is also selected inside the product") {
+			Click(
+				key_fieldName = "Topic",
+				locator1 = "Button#SELECT_BUTTON");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_BODY");
+
+			Click.clickAtNotVisible(
+				key_nodeName = "Topic Category",
+				locator1 = "Treeview#NODE_CHECKBOX");
+
+			SelectFrame(value1 = "relative=top");
+
+			Click(locator1 = "Button#DONE");
+
+			Button.clickPublish();
+		}
+
+		task ("And a new Commerce Product type Display Page Template is created") {
+			JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
+				contentType = "Commerce Product",
+				displayPageTemplateEntryName = "Test Commerce Product Display Page Template",
+				groupName = "Minium");
+		}
+
+		task ("When the Heading fragments are deployed on the page and the fields are mapped") {
+			CommerceDisplayPageTemplates.mapFragmentsToFields(
+				displayPageName = "Test Commerce Product Display Page Template",
+				fieldsList = "Minium,Topic,Test Vocabulary",
+				fragmentName = "Heading",
+				siteURLKey = "minium");
+		}
+
+		task ("And the template is published and Marked as Default") {
+			LexiconEntry.gotoEntryMenuItem(
+				menuItem = "Mark as Default",
+				rowEntry = "Test Commerce Product Display Page Template");
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("And the buyer logs in") {
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("Then in the product details, the buyer can see mapped all selected categories related to these 3 vocabularies") {
+			CommerceAcceleratorsInitializer.changeProductsSorting(sortingType = "Name Ascending");
+
+			Click(
+				key_productName = "ABS Sensor",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
+
+			for (var categoryName : list "Category 1,Topic Category,Brake System") {
+				AssertElementPresent(
+					key_fragmentText = "${categoryName}",
+					key_id = "element-text",
+					locator1 = "CommerceEntry#ASSERT_FRAGMENTS_TEXT_IN_SF");
+			}
```

Comment:
Use `CommerceDisplayPageTemplates.viewMappedFragmentsText()` instead of these lines. Thanks!